src/main/webapp/app/overview/course-statistics/course-statistics.component.ts

```diff
@@ -267,7 +267,8 @@ export class CourseStatisticsComponent implements OnInit, OnDestroy {
             return 0;
         }
         if (result.resultString && result.resultString.indexOf('failed') !== -1) {
-            return null;
+            // TODO: Is this right?
```

Comment:
@maximilianmeier Can you check if returning 0 is ok?

Reply:
we need the null. i reverted it